src/lib/shared/dialog.ts

```diff
@@ -92,7 +108,7 @@ export interface DialogPhrases {
   /** English: "Returning to the previous page will discard unsaved changes on this page. Are you sure?" */
   "configurationWizardDialog.previousWarningText": SimplePhrase;
 
-  /** English: "This page contains non-visible config variables that need to be set by %{organizationName} before continuing." */
```

Comment:
Were these examples wrong to use `%`?  Or did something else change to switch this from `%` to `$`?

Reply:
The usages in the comments were a mix of % & $ previously; I attempted to go through and standardize all the comments to use `$` as part of this PR.